package.json

```diff
@@ -313,6 +313,7 @@
         "displayName": "Settings (Invoices): Batch group usernames and passwords: view",
         "visible": true,
         "subPermissions": [
+          "settings.invoice.enabled",
```

Comment:
not clear who this permission is connected to `Batch group usernames and passwords: view`